### ISSUE TITLE
Epic Bk 2 Ch 8 Instance Dour Root won't activate

### ISSUE THREAD

**Issue by bcp19** [echoes-emu/echoes-bt#746]
In the instance for the Epic Quest Red Pass, when you get to Dour Root, the Huorn, he doesn't activate. 
You are at: bx150 by118 c0 ox13.0978 oy99.3177 oz332.91 r184 (Instance 2)
Selection: 0x031967600200003B (0x041967600000000B)
 DID: 0x700006A7
 Loc: bx150 by118 c0 ox9.15111 oy97.5258 oz332.676 r52 (Instance 2)
 Move table: 0x070001EE
State stack:
(Idle)
Melee combat mode: combat_monster
Ranged combat mode: combat_monster
KungFu NPC: Yes

**Comment by Stressy23** [comment]
Worked for me 3 days ago. Did radaghast not follow you?

**Comment by woodx319** [comment]
yeah are you sure you didn't have to talk to anybody or anything? Haven't done that quest in 462 years

**Comment by bcp19** [comment]
I was following Radagast and the tree never activated. There's no one to talk to, you're just following Radagast as he meanders through the place talking to the animals and fighting off the things he walks into.

**Comment by bcp19** [comment]
Dour Root activated today when I did the quest, but did not activate yesterday when I made an LM and ran it through the Bk 2 quests. Unless you patched it, it seems a random issue

**Comment by bcp19** [comment]
Re-checked this today, Dour Root activated as normal. but I noticed a couple errors in the scripting. First, when you activate him his dialgue is

Radagast says, "So my friend, what troubles you? I see that this corruption has affected everything."
Radagast says, "I will do my best to help you. Come with me."
**Radagast says, "Lîn bellas gala!"**
Radagast says, "Beware friends, this Dourroot is angered!"

Only the line in Bold is missing

When you get to the Ghost Dannasen. The following dialogue occurs:

Radagast says, "Well met, sir. Where have the Eglain gone?"
Dannasen says, "I can speak not of such matters. Neven bids that I hold my post, no matter what."
Radagast says, "Speak to this shade again, perhaps there is more that he can tell us."
Radagast says, "I must see to the safety of my little friends here."

But the Wiki says Radagst should also say:

Radagast says, "Come, my little friends."

Later, during the fight with Neven you should see the following, but the lines that are bolded are not spoken:

Neven says, "Your presence is unwelcome, but amusing. What did you hope to accomplish, I wonder?"
Neven says, "It will not matter...soon I shall claim your bones for Ivar and for the Red-maid."
Neven says, "Arise, brethren, there is life to claim!"
**Vile Wight says, "Angmar's army shall stop you!"**
Neven says, "To me! Slay them!"
**Vile Wight says, "Ivar will avenge me!"**

When Neven is defeated and you get Dannasen to go into action, you follow him and run into a trio of Wights and the following should be said:

**Wretched Wight says, "Ivar has warned us about people such as you!"
Wretched Wight says, "Angmar's army shall stop you!"**
Wretched Wight says, "Ivar has told us that none shall pass this place! Slay them!"
**Radagast says, "Come to me, my friends!"**

But again, the lines that are bolded are not present.

When you get to Elsa the Bold the following dialogue occurs:

Radagast says, "Ah, Elsa...and Eriac! I had feared the worst, but you seem hale."
**Radagast says, "Of course I have come, child!"
Elsa the Bold says, "Radagast! You have come!"**
Radagast says, "And you should not have been so hasty in coming here! I told you it was folly!"
Elsa the Bold says, "You were right... I shall know better for the future!"
Radagast says, "Do not feel too badly, it was a great feat to come as far as you have!"
Radagast says, "<[name]>, all of you, I sense our journey is nearly at an end."
Radagast says, "Rest now and speak to Elsa to learn what you may of her battles."
Radagast says, "Her insight may prove helpful in the fight ahead."

But the lines that are bolded should be reversed.

After you talk to Elsa, Radagast says:

Radagast says, "Bring your rest to its end, for we must hurry and continue on our journey."

Which he is supposed to say after you have talked to him, not before.

Throughout the entire instance, Radagast never attacks anything, which is normal according to the Wiki, but you see him perform animations where he is surrounded by a blue or green aura briefly. 

Quest giver: Radagast

Selection: 0x03192740000009D4 (0x041927400000001C)
 DID: 0x7000067C
 Loc: bx146 by116 c3 ox5.43191 oy25.1493 oz-81.5343 r172 (Instance 0)
 Move table: 0x0700003B
State stack:
(Idle)
Melee combat mode: combat_monster
Ranged combat mode: combat_monster

Inside Instance:

Radagast:

Selection: 0x0319676003000048 (0x0419676000000041)
 DID: 0x70008ACA
 Loc: bx150 by118 c0 ox98.1815 oy91.9893 oz334.259 r107 (Instance 3)
 Move table: 0x0700003B
State stack:
(Idle)
Melee combat mode: combat_monster
Ranged combat mode: combat_monster
KungFu NPC: Yes

Dannasen:

Selection: 0x031957600300006F (0x0419576000000001)
 DID: 0x700006A3
 Loc: bx149 by118 c0 ox49.5299 oy64.6658 oz336.539 r64 (Instance 3)
 Move table: 0x070000CF
State stack:
(Idle)
Melee combat mode: combat_monster
Ranged combat mode: combat_monster

Elsa: 

Selection: 0x0319677003000037 (0x041967700000000C)
 DID: 0x700006A4
 Loc: bx150 by119 c0 ox54.8265 oy50.5292 oz344.886 r250 (Instance 3)
 Move table: 0x070001BD
State stack:
(Idle)
Melee combat mode: combat_monster
Ranged combat mode: combat_monster

**Comment by bcp19** [comment]
This quest cannot be canceled after you have acquired it. I was going to cancel it in order to run the instance a second time to double check some things, but the selection is not highlighted and unselectable.
